lib/yabeda/sidekiq.rb

```diff
@@ -33,6 +33,7 @@ module Sidekiq
       gauge     :jobs_dead_count,      tags: [],        comment: "The number of jobs exceeded their retry count."
       gauge     :active_processes,     tags: [],        comment: "The number of active Sidekiq worker processes."
       gauge     :queue_latency,        tags: %i[queue], comment: "The queue latency, the difference in seconds since the oldest job in the queue was enqueued"
+      gauge     :worker_runtime,       tags: %i[queue worker], comment: "The actual worker job runtime"
```

Comment:
job means the exact job, it case we will return the maximum of all runing jobs with the same kind, `job_runtime` sounds weird
I think it will be better to drop the worker tag and have only the queue dimension and name the gauge `queue_max_runtime`